src/test/java/com/example/demo/CheckHTTPResponse.java

```diff
@@ -0,0 +1,26 @@
+package com.example.demo;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.test.web.server.LocalServerPort;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```

Comment:
Do we actually need this RANDOM_PORT here?

Reply:
I was reading the documentatioin from https://spring.io/guides/gs/testing-web/ and saw that if we are using **TestRestTemplate** we need to initialize the port with a random port 
![image](https://user-images.githubusercontent.com/72629723/182649163-2e4499ef-32af-47b3-96d9-e6671907ec13.png)
If we use **MockMvc** it is not needed to star a port
![image](https://user-images.githubusercontent.com/72629723/182649376-ed8121ec-12b3-4383-a6a4-c71b7cb690e6.png)